csi/deployment.go

```diff
@@ -67,6 +67,8 @@ func NewAttacherDeployment(namespace, serviceAccount, attacherImage, rootDir str
 			"--timeout=1m50s",
 			"--leader-election",
 			"--leader-election-namespace=$(POD_NAMESPACE)",
+			"--kube-api-qps=50",
+			"--kube-api-burst=100",
```

Comment:
NIT: Use constants rather than hard-coded values

Reply:
Thank you. Fixed